.pylintrc

```diff
@@ -0,0 +1,50 @@
+[MASTER]
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=CVS
+
+[MESSAGES CONTROL]
+#enable=
+
+disable =
+    # C0301: Line too long
+    # C0330: Wrong hanging indentation before block
+    # disabled because black handles this
+    C0301,C0330,
+
+    # C0115: Missing class docstring
+    # C0114: Missing module docstring
+    C0115,C0114,
+
+    # All convention and refactor for now
+    C,R,
+
+    # W1201: Specify string format arguments as logging function parameters
+    # W1202: Use % formatting in logging functions and pass the % parameters as arguments
+    W1201,W1202,
+
+    # W0612: Unused variable
+    # W0613: Unused argument
+    W0612, W0613,
```

Comment:
Might want to look at the unused variables. Unused arguments are annoying for abstract methods.